packages/ts/generator-typescript-plugin-backbone/test/name-clash/NameClash.spec.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import snapshotMatcher from '@hilla/generator-typescript-utils/testing/snapshotMatcher.js';
+import { expect, use } from 'chai';
+import sinonChai from 'sinon-chai';
+import BackbonePlugin from '../../src/index.js';
+import { createGenerator, loadInput } from '../utils/common.js';
+
+use(sinonChai);
+use(snapshotMatcher);
+
+describe('BackbonePlugin', () => {
+  context('when primitive types are used', () => {
```

Comment:
Probably, this description should have something about name clash?

Reply:
Yes, I forgot that